### PR TITLE
Use CMD instead of ENTRYPOINT

### LINKED ISSUE #76: ENTRYPOINT should be CMD
The image should allow me to run any arbitrary command without starting up Chia itself, but since it's an entry point I cannot use [docker run](https://docs.docker.com/engine/reference/commandline/run/) with a command
https://github.com/Chia-Network/chia-docker/blob/97a13dfdd785d0987709a76fdf9de4ac0cf42595/Dockerfile#L28

### PR DESCRIPTION
**Pull request by davidbarratt**
Fixes #76

### CODE REVIEW

**Review by shamaevnn — APPROVED** [review]

**Comment by justinengland** [comment]
branch has conflicts. also, not 100% sure I agree with the idea that you must be able to run esoteric commands. Docker exec can drop you to shell on the container.

**Comment by davidbarratt** [comment]
> branch has conflicts. also, not 100% sure I agree with the idea that you must be able to run esoteric commands. Docker exec can drop you to shell on the container.

I'll fix the conflicts. `docker exec` only works if it's currently running. What if I want to generate plots instead of startup a farmer? Should I just be able to execute the relevant `chia` command?

**Comment by justinengland** [comment]
i would say that plotting from the container is at best a secondary concern. Not really a use case Id like to bake into the default setup. 

its easy enough imo to get into the container run time to start plotting. 

its a fair point. let me think through the orchestration changes I would need to make for GHA.

File: docker-start.sh [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    exit
+  else
+    chia configure --set-farmer-peer ${farmer_address}:${farmer_port}
+    chia start harvester
+  fi
+else
+  chia start farmer
+fi
+
+tail -f ~/.chia/mainnet/log/debug.log
```
Comment (cmmarslender) [inline docker-start.sh:15]:
```suggestion
tail -f $CHIA_ROOT/log/debug.log
```

This file doesn't necessarily exist right away either - I just tested a build of this locally (with latest main merged in) and I'm getting:

```
...
Daemon not started yet
Starting daemon
chia_harvester: started
chia_farmer: started
chia_full_node: started
chia_wallet: started
tail: cannot open '/root/.chia/mainnet/log/debug.log' for reading: No such file or directory
tail: no files remaining
```

Need to either go back to the previous `while true; ...` stuff, or else, maybe just `touch $CHIA_ROOT/log/debug.log` before trying to tail it

**Review by cmmarslender — CHANGES_REQUESTED** [review]
Looking through PRs today, and I think I do like how the accomplishes your goal of supporting chia commands without necessarily having chia running - Its a bit out of date with main, but if you can get it updated with latest changes from main and take a look at the proposed change below (and the associated problem with that `tail`) I can do some testing on the container this week to make sure it still works for all the existing cases.

**Comment by cmmarslender** [comment]
Closing in favor of #110 (includes these changes plus a few fixes and resolving conflicts)

**Pull request closed without merging**